charts/ccsm-helm/charts/operate/templates/service.yaml

```diff
@@ -3,6 +3,13 @@ kind: Service
 metadata:
   name: {{ include "operate.fullname" . }}
   labels: {{- include "operate.labels" . | nindent 4 }}
+  annotations:
+    {{- if .Values.global.annotations}}
+    {{- toYaml  .Values.global.annotations | nindent 4 }}
+    {{- end }}
+    {{- if .Values.service.annotations}}
+    {{ .Values.service.annotations | toYaml | nindent 4 }}
+    {{- end }}
```

Comment:
:wrench: Lets use one way to apply the values. You can decided whether `toYaml Value` or via pipe `value | toYaml`